src/Documentation/2.0/Migration1.5.md

```diff
@@ -12,3 +12,53 @@ The bulk of the Orleans APIs stayed unchanged in 2.0 or implementation of those
+```
+        public static ILoggingBuilder AddMessageBulkingLoggerProvider<TDecoratedLoggerProvider>(this ILoggingBuilder builder, TDecoratedLoggerProvider provider, EventBulkingOptions eventBulkingOptions = null)
+            where TDecoratedLoggerProvider : ILoggerProvider
+```
+This method would apply message bulking feature to any `TDecoratedLoggerProvider` you choose.
+			
+If you still want to use `ILogConsumer`, with message bulking feaure alone side, you need to configure it through `ILoggingBuilder` as well. The support for `ILogConsumer` lives in `Orleans.Logging.Legacy` package. So you need to add dependency on that package first, and then configure Log consumers through extension method below: 
+```
+public static ILoggingBuilder AddLegacyOrleansLogging(
+            this ILoggingBuilder builder,
+            IEnumerable<ILogConsumer> logConsumers,
+            OrleansLoggerSeverityOverrides severityOverrides,
+            IPEndPoint ipEndPoint = null,
+            EventBulkingOptions eventBulkingOptions = null)
+```
+There's native `AddLogging` method on `IServiceCollection` provided by ASP.NET for you to configure [`ILoggingBuilder`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.loggingservicecollectionextensions.addlogging?view=aspnetcore-2.0#Microsoft_Extensions_DependencyInjection_LoggingServiceCollectionExtensions_AddLogging_Microsoft_Extensions_DependencyInjection_IServiceCollection_System_Action_Microsoft_Extensions_Logging_ILoggingBuilder). We also wrap that method under extention method on `ISiloBuilder` and `IClientBuilder`. So you can call `AddLogging` method on silo builder and client builder as well to configure ILoggingBuilder. 
+below is an example:
+```
+            var severityOverrides = new OrleansLoggerSeverityOverrides();
+            severityOverrides.LoggerSeverityOverrides.Add(typeof(MyType).FullName, Severity.Warning);
+            siloBuilder.AddLogging(builder => builder.AddLegacyOrleansLogging(new List<ILogConsumer>()
+            {
+                new LegacyFileLogConsumer($"{this.GetType().Name}.log")
+            }, severityOverrides));
+```
+ 
+ Since we are going to eventually deprecate and remove LogConsumer feature support in the future, we highly encourage you to migrate off this feature as soon as possible. There's couple approaches you can take to migrate off. One option is to maintain your own `ILoggerProvider`, which creates `ILogger` who logs to all your existing log consumers. This is very similar to what we are doing in `Orleans.Logging.Legacy` package. You can take a look at `LegacyOrleansLoggerProvider` and borrow logic from it. Another option is replace your `ILogConsumer` with existing implementation 
+ of `ILoggerProvider` on nuget which provides identical or similar functionality, or implemente your own `ILoggerProvider` which fits your specfic logging requirement. And configure those ILoggerProviders with ILoggingBuilder.
```

Comment:
This should be the ideal approach, so please put it first. The other one doesn't fully migrate off `ILogConsumer`.
Also, typo in *implemente*